src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -422,8 +430,8 @@ def _start_worker_agent(self) -> None:
         cmd_result = self.send_command(
             " ; ".join(
                 [
-                    "echo Waiting 20s for the agent service to get started",
-                    "sleep 20",
+                    "echo Waiting 30s for the agent service to get started",
+                    "sleep 30",
                     "echo 'Running running Get-Process to check if the agent is running'",
                     "IF(Get-Process pythonservice){echo 'service is running'}ELSE{exit 1}",
```

Comment:
Could we change this to a `for` loop, so that if the service starts earlier, we don't need to wait the full 30s?

I think something like the following would work
```
    "echo Waiting up to 30 seconds for the agent service to get started",
    "for($i=1; $i -le 30 -and "" -ne $err ; $i++){sleep $i; Get-Process pythonservice -ErrorVariable err}",
    "IF(Get-Process pythonservice){echo 'service is running'}ELSE{exit 1}",
```

Reply:
Updated